lib/convert.js

```diff
@@ -2141,6 +2144,15 @@ module.exports = function convert(config) {
             result.typeAnnotation.range = result.typeAnnotation.typeAnnotation.range;
             break;
 
+        case SyntaxKind.ImportType:
+            Object.assign(result, {
+                type: AST_NODE_TYPES.TSImportType,
+                parameter: convertChild(node.argument),
```

Comment:
I renamed the `argument` to `parameter` since it looks like the naming convention here.